lib/resty/acme/autossl.lua

```diff
@@ -332,10 +333,23 @@ function AUTOSSL.check_renew(premature)
       goto continue
     end
 
+    local domain = deserialized.domain
+    if not AUTOSSL.is_domain_whitelisted(domain, true) then
+      log(ngx_INFO, "domain ", domain, " not in whitelist, skipping")
```

Comment:
```suggestion
      log(ngx_INFO, "domain ", domain, " not in whitelist but exists in storage, skipping renewal")
```

Reply:
since you will actually delete the not whitelisted domain afterwards, it's not always "skipping". so this log line will need to move to after Line 345 and don't show if `AUTOSSL.config.enabled_delete_not_whitelisted_domain` is true.